packages/neos-ui-ckeditor-bindings/src/EditorToolbar/LinkIconButton.js

```diff
@@ -191,7 +191,7 @@ class LinkTextField extends PureComponent {
                     placeholder="Paste a link, or search"
                     displayLoadingIndicator={this.state.isLoading}
                     displaySearchBox={true}
-                    setFocus={true}
+                    setFocus={!Boolean(this.props.hrefValue)}
```

Comment:
`setFocus={!(this.props.hrefValue)}`

Reply:
The linter complains :) and `!` is doing an implicit boolean casting IIRC

Reply:
But it's working fine :)

Reply:
yep will fix